openjdk.test.load/src/test.load/net/adoptopenjdk/stf/ParallelStreamsLoadTest.java

```diff
@@ -0,0 +1,61 @@
+/*******************************************************************************
```

Comment:
@lumpfish - is there a reason why this new `ParallelStreamsLoadTest` needs to be iteration-bsased? In other words, can we convert it to time-based?

Reply:
We could - but.....
As explained in https://github.com/AdoptOpenJDK/openjdk-systemtest/issues/379, the TestParallelStreamOperations tests contains code which limits the number of concurrent running instances of the test to two - so there is no point in running it on more than two threads.  On some systems it takes longer than 5 minutes to run those two instances, on others (not much) less.  So if a 5m test were created using just the two threads, then sometimes the tests would run a small number of times, often just the two (one on each of the two threads).
The way the tests were running prior to the splitting out of the TestParallelStreamOperations and the introduction of the recent 5m variant, the tests were only running once anyway, also as explained in https://github.com/AdoptOpenJDK/openjdk-systemtest/issues/379.  So running the tests just once is all that was need to identify all the issues described in https://github.com/eclipse/openj9/issues/10638, and probably other issues too.
If we want to time-base the tests (since this PR is now merged), please raise a follow on issue.

Reply:
Based on the explanation above, it seems to make sense to run the new `TestParallelStreamOperations` test in iteration-based manner. We can watch it for a while and see how it behaves. 

However, as you mentioned in Slack, we should correct the playlist name for `TestParallelStreamOperations` and remove `_5m` from it.

Reply:
Created https://github.com/AdoptOpenJDK/openjdk-tests/pull/2231 to remove the `_5m` suffixes,